integration/apps/rspec/Dockerfile-ci

```diff
@@ -6,6 +6,6 @@ FROM ${BASE_IMAGE}
 COPY . /vendor/dd-trace-rb
 
 # Install dependencies
-# Setup specific version of ddtrace, if specified.
-ENV DD_DEMO_ENV_GEM_LOCAL_DDTRACE /vendor/dd-trace-rb
+# Setup specific version of datadog, if specified.
+ENV DD_DEMO_ENV_GEM_LOCAL_DATADOG /vendor/dd-trace-rb
```

Comment:
I guess these `dd-trace-rb` paths should be updated too

Reply:
Technically these reflect the repo name; we can change these now (say to `datadog-rb`) if we want, but it would be out of step with the current repo name. I recommend holding off and doing repo renaming changes separately.